docs/source/cookbook/ccdc.rst

```diff
@@ -1,10 +1,475 @@
+    :width: 49%
+    :group: time-series-recipe
+
+.. thumbnail:: ../_images/cookbook/time_series/datepicker.png
+    :title: Select AOI based on EE table
+    :width: 49%
+    :group: time-series-recipe
+
+Sensor selection
+^^^^^^^^^^^^^^^^
+
+After clicking the :guilabel:`NEXT` button in the date selection, the sensor selection pop-up menu will automatically open (1). Here you need to specify the sensor(s) and the bands used for the breakpoint detection. You have the choice between 3 types, :guilabel:`OPTICAL` (including the Landsat and Sentinel-2 missions), :guilabel:`RADAR` (including the Sentinel-1 mission) and :guilabel:`PLANET`, where both daily imagery or monthly basemaps can be used as data input (given you have a valid Planet API key).
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/sensor_selection_overview.png
+    :title: Sensor Selection
+    :width: 100%
+    :group: ccdc-asset-recipe
+
+Optical data
+"""""""""""
+
+CCDC is originally tested on optical Landsat satellites. In SEPAL you have the possibility to select and combine all past and present Landsat missions, including Tier 1 and Tier 2 collections, to run it on decadal-long time-series.
+
+.. warning::
+
+    The inclusion of Tier 2 products as well as Landsat 7 may introduce artifacts due to the reduced quality of the data. For recent short-term time-series it might be better to either select the Landsat-8 or Sentinel-2 mission, which deliver imagery from 2013 and 2015, respectively. This will however reduce the density of observations for the underlying time-series.
+
+.. warning::
+
+    For very cloud-prone regions, it is also possible to combine the Landsat data with the Sentinel-2 data to densify the underlying time-series. Note that because of differences in the sensors (although band names are equal) and the overpass time, artifacts may be introduced that will affect the breakpoint detection.
+
+The breakpoint detection is at the heart of CCDC. The respective selection of bands can considerably affect the outcome of the CCDC breakpoint detection. Unfortunately, there does not seem to be a *"one size fits all"* preset for all kinds of applications. Scientific evidence is suggesting to use all color bands but the blue `Zhu et al 2020 <https://www.sciencedirect.com/science/article/pii/S0034425719301002>`_. According to the study, the selection of additional ratio bands does not add any improvement. However, it should be noted that this assumption is based on the detection of all types of land cover changes and that the study uses a modified version of CCDC (named COLD), where the change in bands are weighted differently than in the original version used in SEPAL, respectively Google Earth Engine.
+
+.. tip::
+
+    The use of the color bands allows you to later select the *Green* and the *Swir1* band as TMASK bands for CCDC's internal, multi-temporal cloud removal. You find this in the :guilabel:`OPT` button pop-up menu under :guilabel:`MORE`.
```

Comment:
could we use the `:code:` role for bands ? (that's the convention I used so far unless you prefer otherwise and I'll change it)

Reply:
ok, haven't noted that so far.

Reply:
because that's undeployed branches ;-)